Source/OxyPlot.Tests/Series/DataRangeTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DataRangeTests.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Tests the <see cref="DataRange" /> struct.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Tests.Series
```

Comment:
I don't think the `.Series` namespace is necessary: other series specific tests are just thrown in `OxyPlot.Tests`, and the `DataRange` is pretty generic anyway.